src/ExpressionEvaluator/Core/Source/ResultProvider/Formatter.Values.cs

```diff
@@ -373,7 +373,6 @@ internal string FormatPrimitive(DkmClrValue value, ObjectDisplayOptions options,
             }
             else
             {
-                Debug.Assert(value.HostObjectValue != null);
```

Comment:
I'll revert this change for now. The assert can be updated when IsFunctionPointer is available. Same with the other assert below.